components/feedstock/links.js

```diff
@@ -13,12 +13,8 @@ export const Links = ({ links, doi_citation }) => {
   doi ? allLinks.push({ label: `doi:${doi}`, href: doi_citation }) : null
 
   return (
-    <Flex sx={{ flexDirection: 'column', gap: 3 }}>
-      <Text sx={{ fontSize: 1, fontWeight: 'bold', color: 'muted' }}>
-        Links:
-      </Text>
-
-      <LinkGroup members={allLinks} sx={{ ml: 4 }} />
-    </Flex>
+    <Box sx={{ mt: 4 }}>
+      <LinkGroup members={allLinks} inverted />
```

Comment:
> * The font-size of the grey links at the bottom is inconsistent with the other metadata values (18px vs 16px). I'd suggest bumping to 18px across the board at this breakpoint.

@katamartin, unless i'm missing something, it appears that the [LinkGroup](https://github.com/carbonplan/components/blob/ccdb1c30f30920b5f2150655e4faa1496ca6128f/src/link-group.js#L6) doesn't propagate the`fontSize` to the children components. Do you have any recommendation as to how to pass the `fontSize` to the children of `LinkGroup`?

Reply:
> The font-size of the grey links at the bottom is inconsistent with the other metadata values (18px vs 16px). I'd suggest bumping to 18px across the board at this breakpoint.

I think this means that you would be making the change not to the `LinkGroup` styles but instead to the other metadata strings.

For future reference, you should be able to achieve customizations not explicitly supported via specific `sx` props by using CSS selectors for descendent elements, e.g., `sx={{ '& a': { fontSize: WHATEVER }}}`. Also, because `LinkGroup` was defined to package up and enforce consistency for link rendering on the CarbonPlan site, it's a very simple component! If we were to deviate from those styles here, it would be totally reasonable to "eject" and just wrap some `Button`s with our own `Flex` or similar.

Reply:
> For future reference, you should be able to achieve customizations not explicitly supported via specific `sx` props by using CSS selectors for descendent elements, e.g., `sx={{ '& a': { fontSize: WHATEVER }}}`.

that's awesome!! thank you for the tip